zap/src/main/java/org/zaproxy/zap/extension/brk/BreakpointsParam.java

```diff
@@ -44,13 +44,27 @@
     private static final String PARAM_UI_BUTTON_MODE = PARAM_BASE_KEY + ".buttonMode";
     private static final String PARAM_BRK_ALWAYS_ON_TOP = PARAM_BASE_KEY + ".alwaysOnTop";
     private static final String PARAM_BRK_IN_SCOPE_ONLY = PARAM_BASE_KEY + ".inScopeOnly";
+    private static final String SHOW_IGNORE_FILES_BUTTONS =
+            PARAM_BASE_KEY + ".ShowIgnoreFilesButtons";
```

Comment:
- Start with lower case.
- Technically this is no longer just about ignore files, scope button is included.